packages/web-components/src/label/README.md

```diff
@@ -0,0 +1,121 @@
+# Label
+
+> A label represents a caption for an item in a user interface.
+
+<br />
+
+## **Design Spec**
+
+[Link to Label Design Spec in Figma](https://www.figma.com/file/jpWO2FMBefirTyThf5Rg2P/Label?node-id=2%3A476&t=QCdofuTbXkUjMS4d-0)
+
+<br />
+
+## **Engineering Spec**
+
+Due to the infeasibility of associating elements across the shadow DOM boundary the `fluent-label` will not have access to the element it is intended to reference. For this reason the `fluent-label` is for visual purposes only.
+
+The fluent-label has several visual font size (small, medium, large) and font weight(regular, semibold) options. The fluent-label also provides appearances for required and disabled states.
+
+### Use Case
+
+Creating a simple label element with an optional info icon and optional required state
+
+<br />
+
+## Class: `Label`
+
+<br />
+
+### **Variables**
+
+<br />
+
+### **Fields**
+
+| Name       | Privacy | Type                           | Default     | Description                          |
+| ---------- | ------- | ------------------------------ | ----------- | ------------------------------------ |
+| `required` | public  | `boolean`                      | `false`     | Specifies required styling for label |
+| `disabled` | public  | `boolean`                      | `false`     | Sets disabled state for label        |
+| `size`     | public  | `"small"` `"medium"` `"large"` | `"medium"`  | Specifies font size for label        |
+| `weight`   | public  | `"regular"` `"semibold"`       | `"regular"` | Specifies font weight for label      |
+
+<br />
+
+### **Methods**
+
+<br />
+
+### **Events**
+
+<br />
+
+### **Attributes**
+
+| Name       | Field    |
+| ---------- | -------- |
+| `required` | required |
+| `disabled` | disabled |
+| `size`     | size     |
+| `weight`   | weight   |
+| `id`       | id       |
```

Comment:
nit, unless this needs to be observable it exists on HTMLElement and probably doesn't need to be enumerated IMO.